package.json

```diff
@@ -38,7 +36,6 @@
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "jsonwebtoken": "^9.0.2",
-    "lint-staged": "^15.2.9",
```

Comment:
[blocking question]  Is there a reason why you are dropping `lint-staged` here?  I'm hoping it's possible to keep it around as part of a pre-commit experience, even if husky isn't what's running it.

If you do want to get rid of it, though, should the `.lintstagedrc.js` also be removed?

Reply:
(update after chatting with Brian) since we've already got `next lint` for linting, this was only being called in combination with husky. We might bring it back later but for now, makes sense to remove it, too.

Reply:
Went ahead and removed `.lintstagedrc.js`, thanks @jduss4!